src/xcode/ENA/ENA/Source/Scenes/TraceLocations/TypeSelection/__tests_/TraceLocationTypeSelectionViewModelTests.swift

```diff
@@ -0,0 +1,114 @@
+////
+// 🦠 Corona-Warn-App
+//
+
+import XCTest
+@testable import ENA
+
+class TraceLocationTypeSelectionViewModelTests: XCTestCase {
+
+	func testGIVEN_SectionNames_THEN_NamesMatch() {
+		// GIVEN
+		let viewModel = TraceLocationTypeSelectionViewModel(
+			[.location: [],
+			 .event: []
+			],
```

Comment:
The formatting is a bit off in these tests, maybe like this?

```suggestion
			[
				.location: [],
				.event: []
			],
```